src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -85,24 +88,20 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
 
     ### Path parameters
 
-    Sometimes you want to name certain parts of the path and access them as parameters.
-
+    When defining an explicit path, you may mark certain parts of the path, to have them available later.
```

Comment:
I think the "to have them available later" could be clarified some. Maybe "you may mark certain parts of dynamic parameters that can be accessed from within the action"?